ci/build-windows.ps1

```diff
@@ -0,0 +1,41 @@
+# !/usr/bin/env powershell
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Stop on errors, similar "set -e" on Unix shells...
```

Comment:
Please change the `...` style comments to statements, for example:

```commandline
# Stop on errors.  This is similar to `set -e` on Unix shells.
```

Reply:
My bad. Didn't notice this earlier.